components/sections/earn-section/index.tsx

```diff
@@ -0,0 +1,225 @@
+        alt="More Possibilities illustration"
+        width={506}
+        height={325}
+        className="absolute -left-[70px] bottom-[82px] z-10 h-[325px] w-[506px] max-w-max sm:left-2 md:bottom-9 md:left-[216px] md:h-[304px] md:w-[473px] lg:bottom-10.5 lg:left-12.5 lg:h-[317px] lg:w-[493px] xl:bottom-[50px] xl:left-18 xl:h-[375px] xl:w-[584px] 2xl:-bottom-[218px] 2xl:left-[113px] 2xl:h-[721px] 2xl:w-[721px]"
+      />
+    ),
+    badgeDescription: "Explore Apps",
+  },
+];
+
+export default function EarnSection() {
+  return (
+    <section className="bg-earn-section mt-14 flex flex-col rounded-3xl pt-4 sm:rounded-4xl md:mt-16 md:pt-12 lg:rounded-3xl lg:pt-4 xl:mt-18 xl:rounded-4xl xl:pt-8">
+      <div className="flex flex-col gap-6 px-2 py-4 sm:p-4 lg:gap-8 2xl:items-start 2xl:gap-14 2xl:p-6">
+        <div className="flex flex-col gap-4 2xl:gap-6">
+          <div className="flex max-w-max items-center justify-center rounded-full bg-[#37D56D40] px-3 py-1 backdrop-blur-xl">
+            <span className="text-sm leading-[22px] text-[#37D56D] lg:text-base lg:leading-[25px]">
+              Put your assets to work
+            </span>
+          </div>
+          <h2 className="bg-earn-headline font-h1 text-3.5xl leading-[38px] -tracking-[0.64px] md:text-4xl md:leading-[43px] md:-tracking-[0.72px] lg:text-5xl lg:leading-[53px] lg:-tracking-[0.96px] xl:text-[56px] xl:leading-[61px] xl:-tracking-[1.12px] 2xl:text-[64px] 2xl:leading-[70px] 2xl:-tracking-[1.28px]">
+            Unlock Your <br /> Earning Potential.
+          </h2>
+        </div>
+        {/**cards stack */}
+        <div className="flex flex-col items-center justify-center gap-2 self-stretch xl:gap-4">
+          {/**first row */}
+          <div className="flex flex-col items-center justify-center gap-2 self-stretch lg:grid lg:grid-cols-3 xl:gap-4">
+            {cards.slice(0, 3).map((cardProps) => (
+              <EarnCard key={cardProps.title} {...cardProps} firstRow />
+            ))}
+          </div>
+          {/**second row */}
+          <div className="flex flex-col items-center justify-center gap-2 self-stretch lg:grid lg:grid-cols-2 xl:gap-4">
+            {cards.slice(3, 5).map((cardProps) => (
+              <EarnCard key={cardProps.title} {...cardProps} />
+            ))}
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+}
+
+interface EarnCardProps extends Card {
+  firstRow?: boolean;
+}
+
+function EarnCard({
+  firstRow,
+  badgeDescription,
+  description,
+  descriptionClassName,
+  iconUri,
+  illustration,
+  link,
+  title,
+}: EarnCardProps) {
+  return (
+    <div
+      className={cn(
+        "relative flex flex-grow flex-col justify-between self-stretch overflow-hidden rounded-2xl border border-solid border-[#3C356D] bg-[#201b43] p-4 xl:rounded-3xl 2xl:p-6",
+        {
+          "h-[330px] md:h-[280px] lg:h-[330px] 2xl:h-[440px]": firstRow,
+          "h-[330px] md:h-[280px] lg:h-[284px] 2xl:h-[330px]": !firstRow,
```

Comment:
"h-[330px] md:h-[280px]" overlap in both of the conditions, we can just move them to the main className, they're redundant